src/Presentation/Factory/EventFactory.php

```diff
@@ -111,7 +113,7 @@ protected function getProperties(Event $event): Generator
     protected function getComponents(Event $event): Generator
     {
         yield from array_map(
-            fn (Alarm $alarm) => $this->alarmFactory->createComponent($alarm),
```

Comment:
The code style was changed here. That is one of the reasons, why the CI pipeline fails.

You can fix the code style by running `make fix` locally.